modules/nf-core/presto/filterseq/main.nf

```diff
@@ -0,0 +1,47 @@
+process PRESTO_FILTERSEQ {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::presto=0.7.1"
```

Comment:
Should this be `conda "${moduleDir}/environment.yml"` so that it uses the environment file?

Reply:
ah yes, good catch!